server.ts

```diff
@@ -627,6 +627,17 @@ function start() {
   }
 }
 
+/**
+ * Check if SSR should be skipped for url or path
+ *
+ * @param url
+ * @param path
+ * @param blacklist
+ */
+function isBlacklistedPathForSsr(url: string, path: string, blacklist: RegExp[]): boolean {
```

Comment:
We should remove the term "blacklist" from our code, as it can have [negative connotations](https://www.splunk.com/en_us/blog/learn/blacklist-whitelist-inclusivity.html).

Please change the terms in your code to either say "exclusion list" or "block list" or similar.  Those terms are more descriptive as well.  So, this method could be named `isExcludedFromSsr()` and the last param could be called `exclusionList` or `excludeRegex` or similar.